app/views/mortgage_calculator/stamp_duties/_stamp_duty_to_pay_panel.html.erb

```diff
@@ -13,16 +13,19 @@
   </div>
 
   <p ng-hide="js" class="stamp-duty__results-tax-rate"><%= I18n.t("stamp_duty.results.sentence", percentage: number_to_percentage(@stamp_duty.percentage_tax, precision: 1)) %></p>
-
   <p class="rendered-from-js stamp-duty__results-tax-rate"><%= I18n.t("stamp_duty.results.sentence_prefix") %> {{ stampDuty.percentageTax() | number: 1 }}<%= I18n.t("stamp_duty.results.sentence_suffix") %></p>
 
-  <p class="stamp-duty__FTB_conditional"><%= I18n.t("stamp_duty.results.FTB_conditional", amount: ftb_threshold) %></p>
+  <% if outside_first_time_rate? %>
+    <p ng-hide="js" class="stamp-duty__FTB_conditional ng-hide is-active"><%= I18n.t("stamp_duty.results.FTB_conditional", amount: ftb_threshold) %></p>
+  <% end %>
+  <p class="rendered-from-js stamp-duty__FTB_conditional"><%= I18n.t("stamp_duty.results.FTB_conditional", amount: ftb_threshold) %></p>
 
```

Comment:
Can some of these very long lines be formatted to read more easily?

Reply:
@margOnline ♻️ https://github.com/moneyadviceservice/mortgage_calculator/pull/319/commits/fff7b763ca184df8f14e43ca1e04113f046e3123